pkg/services/live/live.go

```diff
@@ -403,7 +404,14 @@ func (g *GrafanaLive) handlePushScope(_ *models.SignedInUser, namespace string)
 func (g *GrafanaLive) handleDatasourceScope(user *models.SignedInUser, namespace string) (models.ChannelHandlerFactory, error) {
 	ds, err := g.DatasourceCache.GetDatasourceByUID(namespace, user, false)
 	if err != nil {
-		return nil, fmt.Errorf("error getting datasource: %w", err)
+		// the namespace may be an ID
```

Comment:
is this a temporary or persistent thing?

Reply:
We can remove it when we fix #32853